Cargo.toml

```diff
@@ -110,6 +110,8 @@ object.debug = 0
 rustc-std-workspace-core = { path = 'library/rustc-std-workspace-core' }
 rustc-std-workspace-alloc = { path = 'library/rustc-std-workspace-alloc' }
 rustc-std-workspace-std = { path = 'library/rustc-std-workspace-std' }
+object = { git = 'https://github.com/gimli-rs/object.git' }
+cc = { git = 'https://github.com/dpaoliello/cc-rs.git', branch = 'arm64ec' }
```

Comment:
Requires <https://github.com/rust-lang/cc-rs/pull/910> - which is predicated on `arm64ec` becoming a new `target_arch`.